src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1044,18 +1044,18 @@ private void OnAsyncTaskStateChanged(DynamoScheduler sender, TaskStateChangedEve
 
                         if (Logging.Analytics.ReportingAnalytics)
                         {
-                            var modifiedNodes = "";
                             if (updateTask.ModifiedNodes != null && updateTask.ModifiedNodes.Any())
                             {
-                                modifiedNodes = updateTask.ModifiedNodes
-                                    .Select(n => n.GetOriginalName())
-                                    .Aggregate((x, y) => string.Format("{0}, {1}", x, y));
+                                // Send analytics for each of modified nodes so they are counted individually
+                                foreach (var node in updateTask.ModifiedNodes)
+                                {
+                                    // Notice the executionTimeSpan is still for the whole delta computation
+                                    Dynamo.Logging.Analytics.TrackTimedEvent(
+                                        Categories.Performance,
+                                        e.Task.GetType().Name,
+                                        executionTimeSpan, node.GetOriginalName());
+                                }
```

Comment:
This doesn't seem like a good indicator of performance. How about only tracking when there is only 1 node being modified instead? That way we get less events but at least the info we get is closer to reality.